src/lib/holocene/menu/menu-item.svelte

```diff
@@ -37,6 +40,7 @@
   export let selected = undefined;
   export let destructive = false;
   export let disabled = false;
+  export let disabledText: string = 'unavailable';
```

Comment:
I would leave out the default text here (or set it to an empty string). I think with i18n we want to avoid having any text in Holocene components themselves.

Reply:
Also, maybe we want to make the `Tooltip` optional? If `disabledText` isn't assigned we could just not have the `Tooltip`.